docs/UserGuide.md

```diff
@@ -222,20 +222,20 @@ This command updates the bill of the guest at the specified `INDEX`.
 Depending on the sign ('+' or '-') of your input, the existing bill value be incremented 
 or decremented respectively by the value of your input.
 
-#### Format: `bill INDEX b/BILL`
+**Format**: `bill INDEX b/BILL`
 
-#### Constraints of parameters:
+**Constraints of parameters**:
```

Comment:
Would it be nicer if "P" of parameters is capitalised? If you feel it's fine, feel free to ignore this!

Reply:
I think it will stick with this current one as all the other header/ subheaders are also not capitalised. Thanks for the suggestion though!